docs/content/en/docs/user-guide/configuration-reference.md

```diff
@@ -286,6 +286,7 @@ spec:
 | expected | [AnalysisExpected](/docs/user-guide/configuration-reference/#analysisexpected) | The expected query result. | Yes |
 | interval | duration | Run a query at specified intervals. | Yes |
 | failureLimit | int | Acceptable number of failures. e.g. If 1 is set, the `ANALYSIS` stage will end with failure after two queries results failed. Defaults to 1. | No |
+| skipNoData | bool | If true, it considers as a success when no data returned from the analysis provider. Defaults to false. | No |
```

Comment:
to be honest, this name is good to me but it may confuse our users 🤔 I think about `skipOnNoData` or the name you mentioned previously `ignoreNoData` 👀 What do you think?

Reply:
cc @nghialv 👀

Reply:
@khanhtc1202 Yeah, I'm still debating on that. Seems like we need to spend a little bit more time on this one than I thought.

Reply:
🤔  `skipOnNoData` looks good.